components/login/RegistrationForm.js

```diff
@@ -0,0 +1,28 @@
+function RegistrationForm({ handleRegistration, setUser, setPwd }) {
+  function submitRegistration(event) {
+    event.preventDefault();
```

Comment:
Got an error message that complained about event.preventDefault. tried removing it and it work fine without.

Reply:
Resolved